Header header
float32 mag_gyroscope
float32 mag_accelerometer
float32 mag_magnetometer
